src/main/java/Duke.java

```diff
@@ -1,10 +1,61 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
 public class Duke {
```

Comment:
not compulsory but would be good if you can rename the class according to the name of your chatbot